sdks/go/pkg/beam/core/graph/edge.go

```diff
@@ -163,11 +163,22 @@ type MultiEdge struct {
 	Output []*Outbound
 }
 
-// ID returns the graph-local identifier for the scope.
+// ID returns the graph-local identifier for the edge.
 func (e *MultiEdge) ID() int {
 	return e.id
 }
 
+// Name returns a not-necessarily-unique name for the edge.
```

Comment:
It's not clear to me as a user of this API what I'm supposed to do with this info.

Reply:
Know that the name isn't unique, which is a requirement in various contexts. The tree.go code precisely helps make these transform names unique.